CRM/Odoosync/Upgrader.php

```diff
@@ -79,9 +79,10 @@ private function deleteCustomGroup($customGroupName) {
   }
 
   /**
-   *  Update contact custom data
+   *  Initialize contacts sync information
+   *  Set custom field "action_to_sync" and "sync_status" default values
    */
-  private function updateContactCustomData() {
+  private function initializeContactsSyncInformation() {
```

Comment:
sorry I forget to mention this before, instead of using values directly (e.g 1,1) in the query below directly you should fetch the values from the API instead then use them in the query.